avatars/lib/saferound_test.py

```diff
@@ -0,0 +1,116 @@
+"""Original source: https://github.com/cgdeboer/iteround."""
```

Comment:
Code taken from iteround package and adapted to use pytest rather than unittest -> should not need review